reference-implementation/lib/transform-stream.js

```diff
@@ -134,9 +139,11 @@ function TransformStreamTransform(transformStream, chunk) {
 
   transformStream._transforming = true;
 
+  const transformer = transformStream._transformer;
   const controller = transformStream._transformStreamController;
-  const transformPromise = PromiseInvokeOrNoop(transformStream._transformer,
-                             'transform', [chunk, controller]);
+
+  const transformPromise = PromiseInvokeOrFallback(transformer, 'transform', [chunk, controller],
+                             TransformStreamDefaultTransform, [chunk, controller]);
```

Comment:
Indentation here is odd. Indentation in this file doesn't seem particularly consistent, but I would prefer the second line to be indented after the `(` on the first line.